instant.ts

```diff
@@ -74,6 +74,59 @@ async function runTests(path: string) {
   }
 }
 
+const orderPackageIds = (allPackages, chosenPackageIds) => {
```

Comment:
If the `chosenPackageIds` have dependencies that aren't explicitly lsited in the `chosenPackageIds` then we should load those into the list. This way we can choose a single package and the script can ensure that all its dependencies are added in.

Reply:
addressed in https://github.com/openhie/instant/pull/63/commits/1083e15e0136b5525bc7033b608abdc4c6f3a3df